internal/vulnstore/updater.go

```diff
@@ -31,6 +31,10 @@ type Updater interface {
 	// updater.
 	GetLatestUpdateRef(context.Context) (uuid.UUID, error)
 	// DeleteUpdateOperations removes an UpdateOperation.
+	// A call to GC must be ran after this to garbage collect vulnerabilities associated
```

Comment:
```suggestion
	// A call to GC must be run after this to garbage collect vulnerabilities associated
```

Pretty sure past-tense is incorrect there.